changelog/fragments/scorecard-suppress-events.yaml

```
@@ -0,0 +1,9 @@
+entries:
+  - description: >
+      Updated the sample kuttl-test.yaml scorecard configuration
+      file to specify 'suppressLog: event' which disables kuttl
+      from logging k8s events and allows for RBAC priviledge descalation.
+
+    kind: "change"
```

I think we can consider this a bugfix since the scaffolded kuttl configuration is intended to work out of the box but does not without this change, correct?
```suggestion
    kind: "bugfix"
```

---

the previous kuttl config still works, but without this suppressLog change, you have to have a 'list/get' priv on the 'events' type in your RBAC setup.  So, not sure if this is a bugfix by that definition.  Since we are not scaffolding kuttl tests yet or the RBAC required it might just be a change, but I'm flexible if you see it the other way.

---

Ah I thought we were scaffolding kuttl config stuff. You are right, we are not yet. "change" is fine with me, and I would go as far to suggest not including a changelog fragment since nothing `operator-sdk` does is changing, just documentation.

---

yep, agree, removed the changelog fragment just now.